src/cpuid.cpp

```diff
@@ -56,7 +56,15 @@ POSSIBILITY OF SUCH DAMAGE.
 
 
 #if TORRENT_HAS_ARM && TORRENT_HAS_AUXV
-#include <sys/auxv.h>
+#include <dlfcn.h>
+static unsigned long int _getauxval(unsigned long int type)
```

Comment:
names starting with an underscore in the global namespace are reserved. please choose a better name

Reply:
in fact, it may be enough to replace the ``static`` with putting this function in the anonymous namespace

Reply:
No problem, do you still want to change the name ?